### PR TITLE
[WIP] Migrate kubernetes-release To Cloud Config

### PR DESCRIPTION
**Pull request by commit-dkp**
Exporting networks, vm_types, and vm_extensions.

### CODE REVIEW

File: templates/k8s-infrastructure-warden.yml [hunk 1]
```diff
@@ -1,74 +1,46 @@
 compilation:
-  cloud_properties: {}
+  network: kubernetes-warden
```
Comment (cnelson) [inline templates/k8s-infrastructure-warden.yml]: I think we can probably nuke all this warden stuff; we inherited it from upstream but it's not used.

WDYT @jmcarp ?

Comment (jmcarp) [inline templates/k8s-infrastructure-warden.yml]: Agreed.

File: templates/k8s-jobs.yml [hunk 2]
```diff
@@ -1,99 +1,153 @@
...
 
 - name: guestbook-example
   lifecycle: errand
-  templates:
-  - {name: guestbook-example, release: kubernetes}
-  - {name: flannel, release: kubernetes}
-  instances: 1
-  resource_pool: errand
+  jobs:
+  - name: guestbook-example
```
Comment (cnelson) [inline templates/k8s-jobs.yml]: wtf do we need this?

Comment (cnelson) [inline templates/k8s-jobs.yml]: Do we need this example job?

**Pull request merged**